helm/scylla-operator/values.yaml

```diff
@@ -12,6 +12,8 @@ image:
 
 # Scylla Operator log level, 0-9 (higher number means more detailed logs)
 logLevel: 2
+# Additional arguments to pass to Scylla Operator container.
+additionalArguments: [ ]
```

Comment:
Nit: for consistency with upstream, we could name it `additionalArgs` (as we pass it to the container's `args` field).

Reply:
renamed